docs/_documentations/che-installinfo.md

```diff
@@ -14,39 +14,101 @@ Install Che to use with Codewind or prepare to use Codewind with an existing Che
+   ```
+
+**Installing on Kubernetes:**
+
+1. Create the `che` namespace if it doesn't already exist: `kubectl create namespace che`
+2. Determine your Ingress domain.
     - Set the `spec.server.ingressDomain` field in the Che `.yaml` file to the Ingress domain.
     - If you're unsure of your Ingress domain, ask your cluster administrator.
-3. Install Che:
-    - On OpenShift, run the following command: `chectl server:start --platform=openshift --installer=operator --che-operator-cr-yaml=codewind-checluster.yaml --che-operator-image=quay.io/eclipse/che-operator:7.5.1`
-    - On Kubernetes, run the following command: `chectl server:start --platform=k8s --installer=operator --domain=<ingress-domain> --che-operator-cr-yaml=codewind-checluster.yaml --che-operator-image=quay.io/eclipse/che-operator:7.5.1`
+3. Generate TLS certificates and keys according to https://www.eclipse.org/che/docs/che-7/setup-che-in-tls-mode-with-self-signed-certificate/#generating-self-signed-certificates_setup-che-in-tls-mode-with-self-signed-certificate
+4. Generate a Kubernetes secret containing the TLS secret and key you generated in the previous set:
+   ```
+   $ kubectl create secret tls che-tls --key=domain.key --cert=domain.crt -n che
+   ```
+5. Generate a Kubernetes secret containing the certificate you generated in step 2:
+   ```
+   $ cp rootCA.crt ca.crt
+   $ kubectl create secret generic self-signed-certificate --from-file=ca.crt -n che
+   ```
+6. Run the following command to install Che: 
+   ```
+   $ chectl server:start --platform=k8s --installer=operator --domain=<ingress-domain> --che-operator-cr-yaml=codewind-checluster.yaml --che-operator-image=quay.io/eclipse/che-operator:7.9.0
+   ```
 
 ## Updating an existing Che installation
 
-If you already have a Che installation, you can update it for Codewind.
+If you already have a Che installation with TLS, you can update it for Codewind.
 
 After creating the Codewind ClusterRole from the [Prerequisites](#prerequisites), run the following command, where `$NAMESPACE` is the namespace that your Che workspaces run in. By default, this namespace is `che`.
 ```
-kubectl apply -f https://raw.githubusercontent.com/eclipse/codewind-che-plugin/0.9.0/setup/install_che/codewind-rolebinding.yaml -n $NAMESPACE
+$ kubectl apply -f https://raw.githubusercontent.com/eclipse/codewind-che-plugin/0.10.0/setup/install_che/codewind-rolebinding.yaml -n $NAMESPACE
+```
+
+## Adding Certificates for Che to your browser
+
+If Eclipse Che was configured with self-signed certificates (such as by following the previous steps for Kubernetes, or if you installed on an OpenShift cluster with self-signed-certificates), you need to add the `ca.crt` for Eclipse Che to your browser. These steps are **not** necessary if you installed Che with publicly signed certificates (such as on OpenShift on IBM Cloud).
+
+### Download the OpenShift Router ca.crt
+If running on OpenShift with self-signed certs, follow these instructions:
```

Comment:
How common is the shortened "cert"? Should we write out "certificates" instead? Would the full word be easier for someone with a screen reader or if we want to translate the documentation?